Sources/BraintreeCore/Analytics/FPTIBatchData.swift

```diff
@@ -37,6 +37,7 @@ struct FPTIBatchData: Codable {
         let payPalContextID: String?
         let timestamp: String
         let tenantName: String = "Braintree"
+        let venmoInstalled: Bool?
```

Comment:
IMO we can remove all of the passing down of this boolean from the feature clients and just do the app installed check here. (it would send for every analytic event, but I think that's fine)

Reply:
So just rewrite what we have in `isVenmoAppInstalled` function of `BTVenmoClient` in `FPTIBatchData`?

Reply:
I was also wondering if there is a performance cost to Venmo install check each time analytic event is created.